galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/IKeyStore.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.framework.spi;
+
+/**
+ * A class to represent a Java Keystore and allow for easy appending and generation
+ * 
+ * @author jamesdavies
+ *
+ */
+public interface IKeyStore {
+	/**
+	 * <p>
+	 * Allows a Base64 encoded PEM cert to be appended to the Keystore
+	 * </p>
+	 * 
+	 * @param cert
+	 * @throws CertificateStoreException
+	 */
+	public void appendPem(String cert) throws CertificateStoreException;
+	
+	/**
+	 * <p>
+	 * Allows a Base64 encoded DER cert to be appended to the Keystore
+	 * </p>
+	 * 
+	 * @param cert
+	 * @throws CertificateStoreException
+	 */
+	public void appendDer(String cert) throws CertificateStoreException;
```

Comment:
Aren't DERs binary?

Reply:
yep, changing to InputStream as youll likely be loading from a bundle resource